shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/util/cache/Record.java

```diff
@@ -0,0 +1,42 @@
+package ca.on.oicr.gsi.shesmu.util.cache;
+
+import java.time.Instant;
+
+import ca.on.oicr.gsi.prometheus.LatencyHistogram;
+import io.prometheus.client.Counter;
+
+/**
+ * A record stored in a cache of some kind
+ *
+ * @param <V>
+ *            the type of cached item
+ */
+public interface Record<V> {
+	public static final Counter staleRefreshError = Counter
+			.build("shesmu_cache_refresh_error",
+					"Attempted to refersh a value stored in cache, but the refresh failed.")
```

Comment:
Attempted to refersh -> Attempted to refresh (and below as well)

Reply:
And in one other file. :cry: